MCR-2445 fix acl-editor css for bootstrap 4

eckstueck opened the pull request:
fixed css and javascript

[Link to jira](https://mycore.atlassian.net/browse/MCR-2445).

eckstueck committed:
MCR-2445 fix acl-editor css for bootstrap 4

fixed css and javascript

kkrebs commented on mycore-acl-editor2/src/main/resources/META-INF/resources/modules/acl-editor2/gui/xml/webpage.xml:
```diff
@@ -60,7 +62,7 @@
                <div id="acle2-elem-per-page" class="">
                  <text i18n="ACLE.labels.access.elemPerPage"/>
-                 <input class="input-sm form-control" id="acle2-elem-per-page-input" type="text" value="10"></input>
+                 <input class="input-sm form-control" id="acle2-elem-per-page-input" type="text" value="100"></input>
```
perhaps we can go up to 20? 100 is a lot ...

eckstueck committed:
MCR-2445 fix acl-editor css for bootstrap 4

- set default elements per page to 20
- save elements per page in local storage
- removed deprecated event.which
